CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file.
 
+## [2.13.0](https://github.com/voxpupuli/beaker-hostgenerator/tree/2.13.0) (2024-05-21)
+
+[Full Changelog](https://github.com/voxpupuli/beaker-hostgenerator/compare/2.12.0...2.13.0)
+
+**Fixed bugs:**
+
+- Update Amazon Linux 2 bindings [\#367](https://github.com/voxpupuli/beaker-hostgenerator/pull/367) ([mhashizume](https://github.com/mhashizume))
```

Comment:
If we keep the bug label, the next release should be a patch release, not a minor release.

Reply:
Makes sense! I've updated the PR.